crates/re_viewer/src/app.rs

```diff
@@ -351,6 +353,7 @@ impl App {
             TimeControlCommand::TogglePlayPause => {
                 time_ctrl.toggle_play_pause(times_per_timeline);
             }
+            TimeControlCommand::Restart => time_ctrl.set_time(0),
```

Comment:
Not sure if setting to 0 will work with all time types?

Reply:
It won't - it should go back to the first time point of the current timeline. You can use `times_per_timeline` with `time_ctrl.timeline()` to figure that out!